PROJECT-TIERS.md

```diff
@@ -0,0 +1,76 @@
+
+### Creating a community project
+
+Obviously anyone with an interest in Sigstore can create their own GitHub repository and begin writing code. If such a repo originator believes that their source code / project should become a Community project, they must complete the following steps:
+
+* Create a `ROADMAP.md` document in the root of their repo describing the intended evolution of the project.  
+* Create a `CONTRIBUTORS.md` document in the root of their repo naming the GitHub users who will be responsible for maintaining the project.  
+* Open an issue on the Sigstore Community Repo describing the project, its benefit, and describing why it’s an appropriate candidate for stewardship as a Community project.  
+* Notify the Sigstore TSC (by opening an issue on [github.com/sigstore/tsc](http://github.com/sigstore/tsc)) that they’d like to discuss their project at an upcoming meeting. A Sigstore TSC member will add it to the meeting agenda.  
+* Attend the meeting of the Sigstore TSC where their project is to be discussed  
+* Sigstore TSC will then discuss the project and its roadmap (in closed session, if desired) and vote on whether to bring it into the GitHub organization as a designated Community project.  
+* The originating maintainer will then transfer the GitHub repository to Sigstore, including having its GitHub configuration managed by the Sigstore TSC
+
+## Criteria for up-leveling from Community to Core
+
+* TSC must vote to move a project from Community to Core  
+* There must be a dedicated maintainer and at least one other person acting as a contributor. Ideally these should be from two different employers.  
+* The project must have had at least X contributions within the trailing Y months  
+* In the case of a language client, it must pass all tests described in the sigstore-conformance repo
+
+## Why would a project want to go from Community to Core?
+
+Core projects enjoy consideration for targeted funding from OpenSSF, Sigstore, or other funding sources. Membership in Core is a signal to the broader OSS community that the project is stable enough for adoption or productization.
+
+# Project lists
+
+## Core
+
+### Services / CLI tools
+
+* [Cosign](https://github.com/sigstore/cosign)  
+* [Fulcio](https://github.com/sigstore/fulcio)  
+* [Rekor](https://github.com/sigstore/rekor)  
+* [Policy Controller](https://github.com/sigstore/policy-controller/)  
+* [Timestamp Authority](https://github.com/sigstore/timestamp-authority)
+
+### Libraries / clients
+
+* [sigstore-conformance](https://github.com/sigstore/sigstore-conformance)
+* [sigstore-go](https://github.com/sigstore/sigstore-go)  
+* [sigstore-js](https://github.com/sigstore/sigstore-js)  
+* [sigstore-java](https://github.com/sigstore/sigstore-java)  
+* [sigstore-python](https://github.com/sigstore/sigstore-python)  
+* [sigstore-ruby](https://github.com/sigstore/sigstore-ruby)  
+* [cosign-installer](https://github.com/sigstore/cosign-installer)  
```

Comment:
this should not be services/ tools?

Reply:
I think `Libraries / clients` is the most obvious place.